src/cls/rgw/cls_rgw.cc

```diff
@@ -2938,9 +2938,7 @@ static int usage_iterate_range(cls_method_context_t hctx, uint64_t start, uint64
   if (ret < 0)
     return ret;
 
-  if (truncated) {
-    *truncated = truncated_status;
-  }
+  *truncated = truncated_status;
```

Comment:
@gregsfortytwo I'm not sure you're reading this one correctly.. we do set it unconditionally, as long as it's not null pointer

Reply:
....
yep.

*facepalm*